.gitignore

```diff
@@ -257,3 +257,6 @@ pub/
 
 #Ignore marker-file used to know which docker files we have.
 .eshopdocker_*
+.devcontainer
```

Comment:
Please open this in codespaces.

1. It fails to load
2. Can you add azd to it?

Thanks!

Reply:
We upgraded the `dotnet 5.0` version of the image in the dockerfile to `6.0` to solve the error when installing `dotnet-ef`. In addition, we added the steps to install `azd` in the dockerfile, and added azure-dev extention in `.devcontainer`.